plugins/server/io.ktor/ktor-sessions/2.0/manifest.ktor.yaml

```diff
@@ -1,5 +1,5 @@
 name: Sessions
-description: Adds supports for sessions, with the payload in the client or the server
+description: Adds supports for persistent sessions via cookies or headers
```

Comment:
supports -> support
via -> through (In docs it's better to avoid latin abbreviations)